go/server.go

```diff
@@ -87,6 +87,27 @@ func getAccessToken(c *gin.Context) {
 	})
 }
 
+// Creates an item add token for initializing Link in normal item add mode.
+// In production, this endpoint should be authenticated.
+func getItemAddToken(c *gin.Context) {
+	itemAddToken := c.PostForm("item_add_token")
```

Comment:
This assignment seems unnecessary as the value assigned to `itemAddToken` is never used in the code from here to when it is assigned the `response.AddToken` value below (in line 103).
Looking at other quickstart examples, which do not use an `item_add_token` form parameter on this endpoint, this might be a copy & paste error from `getAccessToken`
```suggestion
```